app/controllers/users_controller.rb

```diff
@@ -1,14 +1,24 @@
 class UsersController < ApplicationController
   before_action :authenticate_user!
+  before_action :requires_admin
 
   def index
     @users = User.list user_index_params
   end
 
+  def edit
+    @user = User.find params[:id]
+  end
```

Comment:
Ah.  Awesome!  This is my preference to keep these instance variables being defined directly in the action without a before_action call.  :smile: